src/interactivity/onecore/ConIoSrvComm.cpp

```diff
@@ -551,34 +532,32 @@ PVOID ConIoSrvComm::GetSharedViewBase() const
 
 [[nodiscard]] NTSTATUS ConIoSrvComm::InitializeBgfx()
 {
-    NTSTATUS Status;
-
-    Globals& globals = ServiceLocator::LocateGlobals();
+    const auto& globals = ServiceLocator::LocateGlobals();
     FAIL_FAST_IF_NULL(globals.pRender);
-    IWindowMetrics* const Metrics = ServiceLocator::LocateWindowMetrics();
+    const auto Metrics = ServiceLocator::LocateWindowMetrics();
 
     // Fetch the display size from the console driver.
-    const RECT DisplaySize = Metrics->GetMaxClientRectInPixels();
-    Status = GetLastError();
+    const auto DisplaySize = Metrics->GetMaxClientRectInPixels();
+    NTSTATUS Status = GetLastError();

```

Comment:
wow, GLE doesn't even return an NTSTATUS... wait does it?

Reply:
That's a direct translation of the previous code. It compiles fine with `auto` so I changed it.